biz.aQute.tester.test/bnd.bnd

```diff
@@ -0,0 +1,22 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
+
+-exportcontents: *
+
+Bundle-Description: A library to be statically linked. Used for testing tester implementations. \
```

Comment:
I would rather this test project not produce a jar file (`-nobundles: true`), like other *.test projects. Since you need to refer to the classes from other tests in their `-testpath`, just use ` biz.aQute.tester.test;version=project` to reference the bin folder of this project.

Reply:
Fixed.

Reply:
Hit a minor issue with this: see #3341.